lib/improver/utilities/cube_manipulation.py

```diff
@@ -835,6 +835,26 @@ def enforce_coordinate_ordering(
     return cube
 
 
+def enforce_float32_precision(input_cubes):
+    """Take input cube of any precision and convert to float32.
+
+    Args:
+        input_cubes (list of cubes or single cube):
+            List containing one or more iris cubes to test if not float32
+            precision and downscale to float32 if necessary.
```

Comment:
You should mention here something like: `If a list item is not an Iris cube, then this item is skipped.`